src/bptl/work_units/zgw/tasks/zaak.py

```diff
@@ -231,3 +232,61 @@ def perform(self):
             "archiefnominatie": resultaat["archiefnominatie"],
             "archiefactiedatum": resultaat["archiefactiedatum"],
         }
+
+
+@register
+class LookupZaak(ZGWWorkUnit):
+    """
+    Look up a single ZAAK by identificatie and bronorganisatie.
+
+    This task looks up the referenced zaak, and if found sets the zaakUrl as a
+    process variable. If not found, the variable will be empty.
+
+    You can use this to check if the referenced ZAAK does indeed exist, and relate
+    it to other objects.
+
+    **Required process variables**
+
+    * ``identificatie``: identification of the zaak, commonly known as "zaaknummer"
+    * ``bronorganisatie``: RSIN of the source organization for the zaak. The combination
+        of identificatie and bronorganisatie uniquely identifies a zaak.
+    * ``services``: JSON Object of connection details for ZGW services:
+
+        .. code-block:: json
+
+          {
+              "<zrc alias>": {"jwt": "Bearer <JWT value>"},
+          }
+
+    **Optional process variables (Camunda exclusive)**
+
+    * ``callbackUrl``: send an empty POST request to this URL to signal completion
```

Comment:
Why is the callbackUrl needed/handy/used?

Reply:
This was a request from another party looking into using BPTL. They found it not ideal that they'd have to poll the Camunda process instance to check if the action was completed, so they can provide an optional callback URL that will be notified on completion of the Camunda task.

I don't think it's actually used, but it's a generic implementation available for any task running in a Camunda context (i.e. -> not used if you call the task via the REST API).

I'm considering dropping that support in 1.0 because it's not used and because we're using long-polling now that should reduce latency significantly.